lib/install/data/data.go

```diff
@@ -42,7 +42,7 @@ type Data struct {
 	common.Images
 
 	ImageDatastorePath     string
-	VolumeLocations        map[string]string
+	VolumeLocations        map[string]*url.URL
```

Comment:
yeah 😄 this is what constituted a fair amount of change, but it was desperately needed!